content/pages/education/complaint-tool/form.md

```diff
@@ -10,7 +10,7 @@ description: Use your VA education benefits to pay for college or training progr
     <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
       <li><a href="/">Home</a></li>
       <li><a href="/education/">Education</a></li>
-      <li><a aria-current="page" href="/education/complaint-tool/">Opt Out of Information Sharing</a></li>
+      <li><a aria-current="page" href="/education/complaint-tool/">GI Bill® School Complaint Tool</a></li>
```

Comment:
Should this be "Feedback" rather than "Complaint" now?

Reply:
@cehsu Yes, the updated title is GI Bill School Feedback Tool.